### ISSUE TITLE
Add-GSDriveFilePermissions to share public - 'FileID' is ReadOnly property error

### ISSUE THREAD

**Issue by queks** [scrthq/PSGSuite#47]
When using the following command:

`Add-GSDriveFilePermissions -FileId "<Redacted FileID>" -type Anyone -Role Reader`

I get the following error:
```
Add-GSDriveFilePermissions : 'FileId' is a ReadOnly property.
At line:1 char:1
+ Add-GSDriveFilePermissions -FileId "[redacted FileID]"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Add-GSDrivePermission
```

**Comment by scrthq** [comment]
I'm on it! thanks, @queks!!

**Comment by scrthq** [comment]
Fixed in v2.6.1! Update when you're able to and let me know if you're still having issues!

**Comment by queks** [comment]
It works! Thanks!
